src/test/java/io/vertx/junit5/CountingCheckpointTest.java

```diff
@@ -101,4 +103,43 @@ void check_strict_checkpoint() {
       .isInstanceOf(IllegalStateException.class)
       .hasMessage("Strict checkpoint flagged too many times");
   }
+
+  @Test
+  @EnabledIfEnvironmentVariable(named = "SLOW_UNIT_TESTS", matches = ".*")
```

Comment:
Why do use that environment variable and where does it come from?

Reply:
This unit tests takes a few minutes depending on the speed of your hardware.